map_gen/maps/danger_ores/modules/map_poll.lua

```diff
@@ -18,11 +18,21 @@ end)
 local normal_mod_pack = 'normal_mod_pack'
 local bobs_mod_pack = 'bobs_mod_pack'
 local krastorio_mod_pack = 'krastorio_mod_pack'
+local omnimatter_mod_pack = 'omnimatter_mod_pack'
+local bz_mod_pack = 'bz_mod_pack'
+local ei_mod_pack = 'ei_mod_pack'
+local py_short_mod_pack = 'py_short_mod_pack'
+local ir3_mod_pack = 'ir3_mod_pack'
 
 local mod_packs = {
     normal_mod_pack = 'danger_ore29',
     bobs_mod_pack = 'danger_ore_bobs3',
-    krastorio_mod_pack = 'danger_ore_krastorio2'
+    krastorio_mod_pack = 'danger_ore_krastorio2',
```

Comment:
Just bumping the mod pack version.

```suggestion
    krastorio_mod_pack = 'danger_ore_krastorio3',
```